Sources/MongoSwift/ConnectionString.swift

```diff
@@ -7,6 +7,10 @@ internal class ConnectionString {
     /// Tracks whether we have already destroyed the above pointer.
     private var destroyedPtr = false
 
+    /// Minimum possible value for a heartbeatFrequencyMS specified via the URI or options.
+    /// This may be overriden in the tests via an internal option on `MongoClientOptions`.
+    private var minHeartbeatFrequencyMS: Int32 = 500
```

Comment:
I added a patch to our vendored libmongoc to allow setting this number really low for the tests, so I also had to move up the validation of this option to the Swift layer.